cmd/podman/system/connection/shared.go

```diff
@@ -0,0 +1,30 @@
+package connection
+
+import (
+	"bytes"
+
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/ssh"
+)
+
+// ExecRemoteCommand takes a ssh client connection and a command to run and executes the
+// command on the specified client. The function returns the Stdout from the client or the Stderr
+func ExecRemoteCommand(dial *ssh.Client, run string) (string, error) {
+	sess, err := dial.NewSession() // new ssh client session
+	if err != nil {
+		return "", err
+	}
+	defer sess.Close()
+
+	var buffer bytes.Buffer
+	var bufferErr bytes.Buffer
+	sess.Stdout = &buffer                 // output from client funneled into buffer
+	sess.Stderr = &bufferErr              // err form client funneled into buffer
+	if err := sess.Run(run); err != nil { // run the command on the ssh client
+		return "", errors.Wrapf(err, bufferErr.String())
+	}
+	out := buffer.String() // output from command
+	buffer.Reset()
+	bufferErr.Reset()
```

Comment:
Is this necessary? It will be deallocated anyway.

Reply:
good point, will remove. I used to have to do it when the buffer was in the same function twice.